docker/Dockerfile

```diff
@@ -1,6 +1,15 @@
 FROM ubuntu:14.04
 MAINTAINER svanoort <samvanoort@gmail.com>
 
+# User id of the build user
+ARG uid=4242
```

Comment:
I tried to avoid 1000 not to collide with non-system user accounts but common distributions' images do not seem to have any out of the box. So I am fine with that.